.gitignore

```diff
@@ -91,6 +91,8 @@ fabric.properties
 !.vscode/launch.json
 !.vscode/extensions.json
 *.code-workspace
+launch.json
```

Comment:
don't include this in .gitignore as your application usually uses this to read environment variables. e.g. for an application that's deployed, this is where you would store your environment variable so the application knows the environment is development and not production.